### PR TITLE
test: replace babel-code-frame with @babel/code-frame and add missing types to tests

### PR DESCRIPTION
**Pull request by armano2**

### CODE REVIEW

**Comment by JamesHenry** [comment]
Thanks @armano2, some of these changes were already made in https://github.com/JamesHenry/typescript-estree/pull/37 and TypeScript 3.2 was apparently made stable yesterday, so can we merge that first? I'm sorry I didn't realise you were going to work on this stuff tonight or I would have flagged it uo

**Comment by armano2** [comment]
np. i did not realize that those changes are in this other PR

**Comment by JamesHenry** [comment]
Thanks, I'll merge that branch once it's built

File: tests/ast-alignment/fixtures-to-test.ts [hunk 1]
```diff
@@ -10,8 +12,8 @@ interface Fixture {
 interface FixturePatternConfig {
   pattern: string;
   config?: {
-    babylonParserOptions?: any;
-    typeScriptESTreeOptions?: any;
+    babelParserOptions?: BabelParserOptions;
```
Comment (JamesHenry) [inline tests/ast-alignment/fixtures-to-test.ts]: Nice 👍

File: tests/ast-alignment/parse.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ function parseWithTypeScriptESTree(
 interface ASTComparisonParseOptions {
   parser: string;
   typeScriptESTreeOptions?: ParserOptions;
-  babylonParserOptions?: any;
+  babelParserOptions?: any;
```
Comment (JamesHenry) [inline tests/ast-alignment/parse.ts]: We can apply the `BabelParserOptions` here too right?

File: tests/ast-alignment/parse.ts [hunk 3]
```diff
@@ -67,7 +67,7 @@ function parseWithTypeScriptESTree(
 interface ASTComparisonParseOptions {
   parser: string;
   typeScriptESTreeOptions?: ParserOptions;
```
Comment (JamesHenry) [inline tests/ast-alignment/parse.ts]: Arent' these the wrong ones? Coming from `import { ParserOptions } from '@babel/parser';`

File: tests/ast-alignment/fixtures-to-test.ts [hunk 4]
```diff
@@ -370,10 +373,8 @@ let fixturePatternConfigsToTest = [
       'interface-with-all-property-types', // babylon parse errors
       'interface-with-construct-signature-with-parameter-accessibility', // babylon parse errors
       'class-with-implements-and-extends', // babylon parse errors
-      'var-with-definite-assignment', // babylon parse errors
-      'class-with-definite-assignment', // babylon parse errors
       /**
-       * typescript-estree erroring, but babylon not.
+       * typescript-estree throw error, but babylon not.
```
[the remark this answers is not in the document]
Comment (armano2) [reply tests/ast-alignment/fixtures-to-test.ts]: ahh sorry, there was conflict on this line and it mixed it up xd

**Comment by armano2** [comment]
ok i resolved your suggestions xd

-----

i started also on fixing types in converter (as any) xddd and i will split alignment from types to 2 commits

**Comment by JamesHenry** [comment]
Just FYI `feat` would cause a minor version bump, which is not relevant here. This one should be `chore` or `test`

It's not a big deal, as I can amend it before things get merged, but just letting you know

**Comment by JamesHenry** [comment]
More info via the tooling I use: https://github.com/marionebl/commitlint

**Comment by armano2** [comment]
ohh good to know, i was planning to check how you are checking commit messages

**Comment by armano2** [comment]
i found one issue with yarn.lock but i think it should be updated in separated commit

> 8 vulnerabilities found - Packages audited: 32731

**Comment by JamesHenry** [comment]
Awesome, thanks!

**Comment by JamesHenry** [comment]
I'll get this merged now and then I'm signing off for the night. Thanks again for your help!

**Pull request merged**